_episodes/03-create.md

```diff
@@ -226,9 +226,10 @@ draft.txt
 > {: .solution}
 {: .challenge}
 
-Let's tidy up by running `rm draft.txt`:
+Let's tidy up the `thesis` directory by removing the draft we created:
 
 ~~~
+$ cd -		# Return to data-shell/thesis after exercise above
```

Comment:
I'd suggest changing this to a textual note like in the other cases (e.g. "returning to the `data-shell` directory..."). 

Even with the comment (which is great), I think this could trip someone up if:

1. They didn't get to their home directory in one try while doing the exercise, or did some additional moving around before returning to the main lesson.
2. The instructor skipped the exercise (because of time pressure, or whatever other reason)